flareon/src/auth.rs

```diff
@@ -6,6 +6,7 @@
 //!
 //! For the default way to store users in the database, see the [`db`] module.
 
+#[cfg(any(feature = "sqlite", feature = "postgres", feature = "mysql"))]
```

Comment:
Is there a way to create an alias for it? We will repeat that in many places in the code.

One idea I have is to create an empty `__db` feature that each database feature would include as well, and then we could just check for it.

Reply:
Yeah, that's one way to resolve it. Except that I think it should actually just be named `db` (sadly there's no concept of "private" feature flags in Cargo as of now and special naming doesn't change much about this) and we should generate a compilation error in `build.rs` if `db` is enabled, but none of `sqlite`, `postgres`, and `mysql` is (if we don't do that, then `seaquery-binder` will fail to compile as far as I remember). But generally I think this is a decent idea